airflow/www/widgets.py

```diff
@@ -72,3 +72,8 @@ class BS3TextAreaROWidget(BS3TextAreaFieldWidget):
     def __call__(self, field, **kwargs):
         kwargs['readonly'] = 'true'
         return super().__call__(field, **kwargs)
+
+class AirflowModelShowWidget(RenderTemplateWidget):
+    """Airflow model show widget"""
+
+    template = 'airflow/model_show.html'
```

Comment:
What is different in this template from the stock Flask-AppBuilder one?

Reply:
In FAB's template, formatter is invoked and value of the field is passed.
https://github.com/dpgaspar/Flask-AppBuilder/blob/3a6b45b1c12a52a794de27910896cbae61270d6b/flask_appbuilder/templates/appbuilder/general/widgets/show.html#L5

FAB's code is passing value (string) of the field instead of dictionary of key and value, so this code is breaking: https://github.com/apache/airflow/blob/ac93ee863a28d7774477a120696f2430dbb284df/airflow/www/views.py#L3895

above code is expecting dictionary instead of string.

Reply:
This template is specific to just Variables show then? If so we should name it as such.

Reply:
Done